mojaloop/values.yaml

```diff
@@ -1765,7 +1791,9 @@ ml-api-adapter:
                       "client.id": "notification",
                       "group.id": "kafka-ml-api-adapter",
                       "metadata.broker.list": "$kafka_host:$kafka_port",
-                      "enable.auto.commit": false
+                      "enable.auto.commit": false,
+                      "socket.keepalive.enable": true,
+                      "socket.max.fails": 0
```

Comment:
I would suggest also adding the same kafka configs to the "ml-api-adapter" chart:

"socket.keepalive.enable": true
"socket.max.fails": 0